tests/models/api/test_llm.py

```diff
@@ -123,7 +123,12 @@ def test_llm_request(self):
         with self.assertRaises(ValidationError):
             LLMRequest(query=test_query, history=test_history,
                        persona=test_persona, model=test_model, stream=False,
-                       temperature=0.8)
+                       temperature=0.8, best_of=2)
+        # Invalid beam search with best_of=1
+        with self.assertRaises(ValidationError):
+            LLMRequest(query=test_query, history=test_history,
+                       persona=test_persona, model=test_model, stream=False,
+                       temperature=0.8, beam_search=True, best_of=1)
```

Comment:
Need to remove temperature=0.8 here
`# Invalid beam search with best_of=1`
Comment is saing it's testing beam search with best_of=1, and temperature is extra